src/test/resources/ide/language/python/StringLiterals.py

```diff
@@ -0,0 +1,17 @@
+oneTypo = "It is <warning>friend</warning> of human"
```

Comment:
We need to add multilines here :)

Reply:
Do you mean triple quotes? They are testing in DocString.

Reply:
Well, ok)